### PR TITLE
Fixes #37592 - open Documentation in new tab in generic table

### PR DESCRIPTION
**Pull request by MariaAga**

### CODE REVIEW

**Comment by sjha4** [comment]
Is this hasHelpPage flag on the Table used anywhere in any pages? I didn't see any component using it right now to test it.

**Comment by MariaAga** [comment]
It was noticed when reviewing a pr in foreman_ansible, to test I added hasHelpPage in webpack/assets/javascripts/react_app/routes/Models/ModelsPage/index.js

**Comment by sjha4** [comment]
Tested that documentation in action toolbar for the table opens in a new tab with the change. Non-write Ack 👍🏼

File: webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js [hunk 1]
```diff
@@ -173,7 +173,11 @@ const TableIndexPage = ({
     hasHelpPage && {
       title: __('Documentation'),
       icon: <QuestionCircleIcon />,
-      action: { href: customHelpURL || helpURL() },
+      action: {
+        href: customHelpURL || helpURL(),
+        target: '_blank',
+        rel: 'noopener noreferrer',
```
Comment (stejskalleos) [inline webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js]: According to [this answer ](https://stackoverflow.com/a/67839804)on StackOverflow, all browsers handle this automatically.

Comment (MariaAga) [reply webpack/assets/javascripts/react_app/components/PF4/TableIndexPage/TableIndexPage.js]: Good to know, removed!

**Review by jeremylenz — APPROVED** [review]
ACK based on other reviews. :+1:

**Pull request merged**